bazel/v8.patch

```diff
@@ -1,7 +1,8 @@
 # 1. Use already imported python dependencies
 # 2. Disable pointer compression (limits the maximum number of WasmVMs).
-# 3. Add support for --define=no_debug_info=1.
-# 4. Don't expose Wasm C API (only Wasm C++ API).
+# 3. Add support for --define=no_debug_info=1. 
+# 4. Allow compiling v8 on macOS 10.15 to 13.0.
```

Comment:
Would be good to add a todo here for tracking
`TODO(dio) - remove patch when https://bugs.chromium.org/p/v8/issues/detail?id=13428 lands`
or something similar

Reply:
Thanks for the review, @moderation! Sure. Done in abe0ac0.